lib/lib-grid/src/main/java/org/sagebionetworks/repo/model/grid/node/ConstantUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.sagebionetworks.repo.model.grid.node;
+
+import org.json.JSONArray;
+
+public class ConstantUtils {
+
+	/**
+	 * Given a constant value, convert it to JSON by inserting into a JSONArray.
+	 * 
+	 * @param value
+	 * @return
+	 */
+	public static String constantValueToJson(Object value) {
+		if (value == null) {
+			return "[]";
+		}
+		return new JSONArray().put(value).toString();
+	}
```

Comment:
Not necessary to put inside array. But the library we are using may not support non-object/non-array documents. It is likely that our library is following an older version of the JSON spec:

https://stackoverflow.com/a/39352374